packages/platform/sdk/src/endpoints/events/event-types/types.ts

```diff
@@ -0,0 +1,64 @@
+import { z } from "zod";
+
+const CreateEventTypesSchema = z.object({
+  length: z.number().min(1),
+  slug: z.string(),
+  title: z.string(),
+});
+
+const GetEventTypeByIdSchema = z.object({
+  id: z.string(),
+  forAtom: z.boolean().optional(),
+});
+
+export type CreateEventTypeArgs = z.infer<typeof CreateEventTypesSchema>;
+export type GetEventTypeByIdArgs = z.infer<typeof GetEventTypeByIdSchema>;
+
+export type EventType = {
```

Comment:
maybe if we don't reuse types from the monorepo we need to think about adding integration tests for the sdk so that we are informed in CI that the request failed

Reply:
Yes, integration testing is planned. Was talking to Keith about this yesterday.